packages/vue-urql/package.json

```diff
@@ -52,7 +52,7 @@
     "@urql/core": "workspace:*",
     "@vue/test-utils": "^2.3.0",
     "graphql": "^16.0.0",
-    "vue": "^3.2.47"
+    "vue": "^3.4.27"
```

Comment:
This is kind of a breaking change if we require folks to be above 3.4 as our peerDeps hint at > 3

Reply:
Oh right, sorry I was not very careful about that. I will make another PR to resolve the problem with pause only (along with the test and the small typos in the doc).

Reply:
Can you do it in this PR so we don't merge a breaking change

Reply:
Sure (I meant I would make a PR -- or correct this one -- just to correct the issue with pause and leave the update to Vue 3.4).